docs/versionhistory.rst

```diff
@@ -5,6 +5,10 @@ Version history
 
 This library adheres to `Semantic Versioning <http://semver.org/>`_.
 
+**UNRELEASED**
+
+- Fix incorrect return type of raise_from()
```

Comment:
This is the change users will care about, yes?
```suggestion
- Fixed compilation of C extension failing on GCC 14
```

Reply:
Done.